controllers/admin/AdminImportController.php

```diff
@@ -375,15 +375,15 @@ public function __construct()
 
             case $this->entities[$this->trans('Customers', [], 'Admin.Global')]:
                 //Overwrite required_fields AS only email is required whereas other entities
-                $this->required_fields = ['email', 'passwd', 'lastname', 'firstname'];
+                $this->required_fields = ['email', 'lastname', 'firstname'];
```

Comment:
I'm not familiar with the import mechanism, so please bear with me as I try to understand:

here you are making the password not mandatory for customer import...  I saw that @PululuK suggested updating the `CustomerFieldsProvider` class, and indeed in this class the `passwd` EntityField is set as `required`

I guess that you tested that your fix works, do you know when this `CustomerFieldsProvider` class is used, is it possible that it's used for import somewhere else? 🤔

Reply:
This is why I wanted @PululuK to elaborate more on his feedback, I'm not familiar with the class he mentioned. Does it seem that this code is WIP and never used?

Can you help us @PululuK ? :D

Reply:
Hi @kpodemski 
Sorry for the delay bro !
See https://github.com/PrestaShop/PrestaShop/issues/11334